dist/api_data.py

```diff
@@ -55,37 +55,37 @@ def __ge__(self, other):
+            check timestamps are used consistently with the configured
+            \c write_timestamp_usage option for this table, writing
+            an error message if policy is violated. If the library was
+            built in diagnostic mode, drop core at the failing check''',
             choices=['off', 'on']),
         ]),
     Config('verbose', '[]', r'''
-        enable messages for various events. The choices are \c write_timestamp
-        which adds verbose messages as described by \c write_timestamp_usage.
-        Options are given as a list, such as \c "verbose=[write_timestamp]"''',
-        type='list', choices=['write_timestamp']),
+        this option is no longer supported, retained for backward compatibility''',
+        type='list', choices=['write_timestamp'], undoc=True),
     Config('write_timestamp_usage', 'none', r'''
-        describe how timestamps are expected to be used on modifications to
-        the table. This option should be used in conjunction with the
-        corresponding \c write_timestamp configuration under the \c assert and
-        \c verbose options to provide logging and assertions for incorrect
-        timestamp usage. The choices are \c always which ensures a timestamp is
-        used for every operation on a table, \c ordered which ensures that
-        once timestamps are used for a key, they are always used, and also
-        that subsequent updates to each key must use increasing timestamps,
-        \c mixed_mode is like \c ordered except that updates with no timestamp
-        are allowed and have the effect of resetting the chain of updates
-        once the transaction ID based snapshot is no longer relevant, \c
-        never enforces that timestamps are never used for a table and \c
-        none does not enforce any expectation on timestamp usage meaning
-        that no log message or assertions will be produced regardless of the
-        corresponding \c assert and \c verbose settings''', choices=['always',
-        'mixed_mode', 'never', 'none', 'ordered']),
+        describe how timestamps are expected to be used on modifications
+        to the table. This option should be used in conjunction with the
+        corresponding \c write_timestamp configuration under the \c assert
+        and \c verbose options to provide logging and assertions for incorrect
+        timestamp usage. The choices are \c always which ensures a timestamp
+        is used for every operation on a table, \c ordered which ensures
+        that once timestamps are used for a key, they are always used,
+        and also that subsequent updates to each key must use increasing
+        timestamps, \c mixed_mode is like \c ordered except that updates
+        with no timestamp are allowed at any time, \c never enforces that
+        timestamps are never used for a table and \c none does not enforce
+        any expectation on timestamp usage meaning that no log message
```

Comment:
Related to 'verbose' it seems the phrase 'log message' in this description is also meant for the 'verbose' setting.

Reply:
`assert=` configurations, outside of `HAVE_DIAGNOSTIC` builds output error messages, so I think this is OK.